### PR TITLE
Output Watchmaker Version

### PR DESCRIPTION
**Pull request by KevinPlus3**
Adding debug line to output Watchmaker version being run.

Addresses issue #329

### CODE REVIEW

**Comment by lorengordon** [comment]
Probably worth adding `.cache` to `.gitignore`...

**Comment by KevinPlus3** [comment]
Removed .cache directory.  Not sure if I put the lines in the right section as I was playing around with testing at that point.

File: .gitignore [hunk 1]
```diff
@@ -80,9 +80,12 @@ src/*.log
 tests/*.log
 
 # Unit test / coverage reports
+.cache/
```
Comment (lorengordon) [inline .gitignore]: just `.cache` will do it

File: .gitignore [hunk 2]
```diff
@@ -80,9 +80,12 @@ src/*.log
...
 .coverage
 .tox
 .coverage.*
+__pycache__/
+*/__pycache__/
```
Comment (lorengordon) [inline .gitignore]: Is git attempting to include these `__pycache__` directories now? I haven't seen that yet...

**Pull request merged**